CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# How to contribute
+
+If you found a bug or issue, please open a GitHub issue [here](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/issues).
+If you open a PR to fix any issue, please reference the ticket in the PR title.
+A [Symphony SDK team](https://github.com/orgs/SymphonyPlatformSolutions/teams/symphony-sdk/members)
+will have to approve before it is merged and eventually released.
+
+If you want to request an enhancement or feature, please reach out to Symphony support.
```

Comment:
how do we reach the support? (or should we let people use the issues for that too? this way the conversation is public and people can look at already closed issues)